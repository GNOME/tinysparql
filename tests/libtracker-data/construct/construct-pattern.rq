PREFIX foaf: <http://xmlns.com/foaf/0.1/>

CONSTRUCT { ?u a foaf:Person ; foaf:name ?name ; foaf:mbox <mailto:bob@work> } WHERE { ?u a example:A ; example:name ?name }
