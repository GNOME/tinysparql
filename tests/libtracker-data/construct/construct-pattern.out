"a"	"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"	"http://xmlns.com/foaf/0.1/Person"
"b"	"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"	"http://xmlns.com/foaf/0.1/Person"
"c"	"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"	"http://xmlns.com/foaf/0.1/Person"
"d"	"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"	"http://xmlns.com/foaf/0.1/Person"
"a"	"http://xmlns.com/foaf/0.1/name"	"nameA"
"b"	"http://xmlns.com/foaf/0.1/name"	"nameB"
"c"	"http://xmlns.com/foaf/0.1/name"	"nameC"
"d"	"http://xmlns.com/foaf/0.1/name"	"nameD"
"a"	"http://xmlns.com/foaf/0.1/mbox"	"mailto:bob@work"
"b"	"http://xmlns.com/foaf/0.1/mbox"	"mailto:bob@work"
"c"	"http://xmlns.com/foaf/0.1/mbox"	"mailto:bob@work"
"d"	"http://xmlns.com/foaf/0.1/mbox"	"mailto:bob@work"
